pubspec.yaml

```diff
@@ -15,7 +15,7 @@ dependencies:
   flutter:
     sdk: flutter
   cupertino_icons: ^1.0.5
-  wakelock: ^0.6.1+2
+  wakelock_plus: ^1.2.4
```

Comment:
Sorry...
wakelock_plus 1.2.5 is released 4 days ago.
https://pub.dev/packages/wakelock_plus/changelog

Could you update it? 🙏